clients/da-vinci-client/src/main/java/com/linkedin/davinci/kafka/consumer/LeaderFollowerStoreIngestionTask.java

```diff
@@ -183,8 +183,8 @@ public class LeaderFollowerStoreIngestionTask extends StoreIngestionTask {
   private final long newLeaderInactiveTime;
   private final StoreWriteComputeProcessor storeWriteComputeHandler;
   private final boolean isNativeReplicationEnabled;
-  private final String nativeReplicationSourceVersionTopicKafkaURL;
-  private final Set<String> nativeReplicationSourceVersionTopicKafkaURLSingletonSet;
+  private volatile String nativeReplicationSourceVersionTopicKafkaURL;
```

Comment:
Just curious - could you explain why we need volatile here? I think only SIT thread will access this variables (or I am wrong here)